students/Chris_Hallen/lesson03/slicing.py

```diff
@@ -0,0 +1,70 @@
+
+def exchange_first_last(seq):
+    """ this function returns a copy of that sequence with the first and last items exchanged"""
+    beg = seq[slice(len(seq) - 1, len(seq))]
+    mid = seq[slice(1, len(seq) - 1)]
+    end = seq[slice(0, 1)]
+    return beg + mid + end
```

Comment:
you should be using list slicing, no need for slice function: 
`return seq[-1:] + seq[1:-1] + seq[:1]`